.github/workflows/ci.yml

```diff
@@ -118,22 +118,8 @@ jobs:
           - directory
           - tar
           - cpio
-          - gpt_btrfs
+          - gpt_xfs
```

Comment:
Wouldn't `gpt_ext4` make as much sense there then? It might be a bit faster, too.